src/app/products/product.service.ts

```diff
@@ -71,6 +80,11 @@ export class ProductService {
     );
   }
 
+  private _createProduct(product: Product) {
```

Comment:
There should be type
private _createProduct(product: Product): void {

Reply:
Also regarding service, I would add an additional level of abstractions, some kinda facade to communicate between components and service. It should not be so that component gets instance of service directly.
But 90 minutes is not enough for this :)